packages/db-authorization/index.js

```diff
@@ -188,7 +186,7 @@ async function auth (app, opts) {
             return originalFind({ ...restOpts, where, ctx, fields })
           }
           const request = getRequestFromContext(ctx)
-          const rule = findRuleForRequestUser(ctx, rules, roleKey, anonymousRole)
+          const rule = await findRuleForRequestUser(ctx, rules, roleKey, anonymousRole)
```

Comment:
Why `await`? It seems to me that `findRuleForRequestUser` is synchronous

Reply:
it's not anymore